docs/source/basics.rst

```diff
@@ -3,202 +3,147 @@
 Programs and Gates
 ==================
 
-First, initialize a localQVM instance on your laptop. You should have two consoles open in your terminal to run in the
-background.
+.. note::
 
-.. code:: python
-
-    ### CONSOLE 1
-    $ quilc -S
-    port triggered: 6000.
-    [2018-09-19 11:22:37] Starting server: 0.0.0.0 : 6000.
+    If you're running locally, remember set up the QVM and quilc in server mode before trying to use
+    them: :ref:`server`.
```

Comment:
Do you think an xref is enough to help users here? I agree with the sentiment around redundancy but just don't want to leave users confused. Up to you